### DISCUSSION TITLE
Marvel rivals compatibility

### DISCUSSION THREAD

**Discussion by Troopica1** [cafali/SnapKey#39] [General]
Marvel rivals doesn't work, obviously the play test ended now, but in general, games with good anti cheat restrict functionality

**Comment by cafali** [comment]
"games with good anti cheat restrict functionality" can you explain?

Since I did not encounter any problems while testing Overwatch 2, CS2 on Premier and Faceit, or in Valorant.

None of the functionality has been restricted.
